VideoLocker/res/values/strings.xml

```diff
@@ -477,6 +477,14 @@
     <!-- In Sign up screen -->
     <string name="or_sign_up_with_email">or sign up with email</string>
 
+    <!-- New Logistration Launch screen text -->
+    <!-- Button leading user to create account screen -->
+    <string name="launch_register">Register</string>
+    <!-- Button leading user to Find Courses screen from the launch screen -->
+    <string name="launch_discover_courses">@string/find_courses_title</string>
```

Comment:
@marcotuts Just want to confirm that you're saying it's fine if users click "Discover Courses" and then land on screens with the title "Find Courses"? :-( 
I don't want to hold up this PR but offline would like to understand more about why marketing prefers "Discover". Any changes in edx.org will also have some impact on existing documentation.